rpdb/__init__.py

```diff
@@ -8,6 +8,21 @@
 import threading
 import sys
 import traceback
+import exceptions
+
```

Comment:
Please use two new-lines (PEP-8), and also I don't think the `import exceptions` part will work on Python 3, you can probably raise AttributeError directly.